pkg/controller/kibana/config/settings.go

```diff
@@ -73,12 +89,57 @@ func NewConfigSettings(client k8s.Client, kb kbv1.Kibana, versionSpecificCfg *se
 	return CanonicalConfig{cfg}, nil
 }
 
+// getExistingConfig retrieves the canonical config for a given Kibana, if one exists
+func getExistingConfig(client k8s.Client, kb kbv1.Kibana) *settings.CanonicalConfig {
+	var secret corev1.Secret
+	err := client.Get(types.NamespacedName{Name: SecretName(kb), Namespace: kb.Namespace}, &secret)
+	if err != nil && apierrors.IsNotFound(err) {
+		log.V(1).Info("Kibana config secret does not exist", "kibana_namespace", kb.Namespace, "kibana_name", kb.Name)
```

Comment:
Nit: just use `"namespace"` for the key because that's what we've mainly done so far.

```sh
> grep -r Info | grep -o '"[a-z_]*"' | sort | uniq -c | sort -n | grep namespace
      1 "namespaces"
      3 "owner_namespace"
     58 "namespace"
```